application/frontend/src/pages/Standard/standard.scss

```diff
@@ -6,6 +6,10 @@
   &__links-container {
     margin-top: 10px;
   }
+
+  .pagination-container {
```

Comment:
Thank you for making time to have a look at #31 All effort is appreciated, and it does look better!

I have one remark, could you put the centered flexbox back on .pagination-container to center align the pagination again? i.e. add
```
display: flex;
justify-content: center;
```

Reply:
fixed